lang/cs/Org.Apache.REEF.IO/Org.Apache.REEF.IO.csproj

```diff
@@ -57,8 +63,13 @@ under the License.
       <HintPath>$(PackagesDir)\Newtonsoft.Json.$(NewtonsoftJsonVersion)\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="NLog, Version=4.0.0.0, Culture=neutral, PublicKeyToken=5120e14c03d0593c, processorArchitecture=MSIL">
+      <HintPath>..\packages\NLog.4.4.12\lib\net45\NLog.dll</HintPath>
```

Comment:
This should be using $(PackagesDir) notation. as used in the lines above for Newtonsoft. Make this change for all the packages you have included